test/cpp/qps/driver.cc

```diff
@@ -313,11 +313,10 @@ std::unique_ptr<ScenarioResult> RunScenario(
     gpr_log(GPR_INFO, "Starting server on %s (worker #%" PRIuPTR ")",
             workers[i].c_str(), i);
     if (!run_inproc) {
-      servers[i].stub = WorkerService::NewStub(grpc::CreateChannel(
-          workers[i], GetCredentialsProvider()->GetChannelCredentials(
-                          GetCredType(workers[i], per_worker_credential_types,
-                                      credential_type),
-                          &channel_args)));
+      servers[i].stub = WorkerService::NewStub(grpc::CreateTestChannel(
```

Comment:
What's the difference between CreateChannel and CreateTestChannel? I searched for these two APIs and found that most of our tests are using CreateChannel and also the user's codes. The change in driver.cc won't affect the benchmark results, so I just want to make sure qps_worker is using the correct API.

Reply:
Never mind, just figure it out.